pymchelper/shieldhit/detector/estimator.py

```diff
@@ -11,18 +11,18 @@
 class SHEstimator:
     allowed_detectors = {
         SHGeoType.unknown: (),
```

Comment:
This section may need more documentation, perhaps even for each individual estimator.

Reply:
I agree. This is a piece of code which is not used in the conversion process, but to programmatically generate SH12A config files.